pyempaq/main.py

```diff
@@ -157,6 +159,13 @@ def main():
         '-q', '--quiet',
         help="Only events of WARNING level and above will be tracked",
         action="store_const", dest="loglevel", const=logging.WARNING)
+    # if argument passed is -v, execute the function
+    parser.add_argument(
+            '-V', '--version',
+            action='version',
+            version=f"pyempaq {get_version()}")
+
+    # quit after displaying the version
```

Comment:
This comment looks misplaced. That said, the functionality of exiting after showing version is not there.